metrics/ci_integrations/lib/source/buildkite/client_factory/buildkite_source_client_factory.dart

```diff
@@ -23,9 +25,13 @@ class BuildkiteSourceClientFactory
     ArgumentError.checkNotNull(config, 'config');
 
     final authorization = BearerAuthorization(config.accessToken);
+    final headers = {
+      HttpHeaders.userAgentHeader: null,
+    };
```

Comment:
Would it be better to introduce some class with constants to place there this map with the default headers for the CI Integrations tool? So, here and in other places we can just `SomeConstants.defaultHeaders`. Also, can this map be `const`?
```suggestion
    const headers = {
      HttpHeaders.userAgentHeader: null,
    };
```